### PR TITLE
Improve README.md file

### PR DESCRIPTION
**Pull request by SuriyaaKudoIsc**
@drbrain @qrush @dwradcliffe:

Hi again, :smile:

I **improved** the **`README.md`** file. :pencil2:

*Yours,*
[**Suriyaa Kudo**](https://github.com/SuriyaaKudoIsc) :octocat:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,13 +6,13 @@ The official RubyGems blog.
 Setup
 =====
 
-Make sure you have jekyll installed (`gem install bundler; bundle`), and run:
+Make sure you have jekyll installed (`gem install bundler` / `bundle`), and run:
```
Comment (hsbt) [inline README.md]: `gem install bundler; bundle` is oneliner. this change break it.

Comment (SuriyaaKudoIsc) [inline README.md]: What do you mean with `...is oneliner. this change break it.`

Comment (dwradcliffe) [inline README.md]: You can copy/paste this as one command, no need to break this into two parts.

Comment (SuriyaaKudoIsc) [inline README.md]: Ok. :+1:

**Comment by SuriyaaKudoIsc** [comment]
@hsbt @dwradcliffe: This PR can be closed.

**Pull request closed without merging**